src/System.Security.Cryptography.OpenSsl/src/System.Security.Cryptography.OpenSsl.builds

```diff
@@ -3,10 +3,10 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Security.Cryptography.OpenSsl.csproj">
-      <OSGroup>Windows_NT</OSGroup>
+      <OSGroup>Unix</OSGroup>
     </Project>
     <Project Include="System.Security.Cryptography.OpenSsl.csproj">
-      <OSGroup>Unix</OSGroup>
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
Yes after discussing with Jose, I was going to do that - but remove all 1.6 cross-compilation across all three projects. I was planning on deferring that however until just after this PR.